core/system/src/mpu/vmpu.c

```diff
@@ -95,6 +95,10 @@ static int vmpu_sanity_checks(void)
     assert( (uint32_t) __uvisor_config.bss_main_end == (SRAM_OFFSET_START +
                                                         UVISOR_SRAM_LENGTH) );
 
+    /* Verify OS private BSS */
+    assert(__uvisor_config.os_priv_start == __uvisor_config.bss_main_end);
+    assert(__uvisor_config.os_priv_end > __uvisor_config.bss_main_start);
```

Comment:
This should be `__uvisor_config.os_priv_end >= __uvisor_config.os_priv_start` right?

In thiw way the 2 assertions are independent and orthogonal:

* The first one checks the assumption on contiguous sections (uVisor bss followed by os priv mem).
* The second one checks that the os priv mem is a sane section (end >= start).